code/game/machinery/vendors/wardrobe_vendors.dm

```diff
@@ -922,3 +922,38 @@
 				/obj/item/clothing/under/plasmaman/enviroslacks = 60)
 
 	refill_canister = /obj/item/vending_refill/lawdrobe
+
+/obj/machinery/economy/vending/traindrobe
+	name = "\improper TrainDrobe"
+	desc = "This wardrobe dispenses the attire of Nanotrasen's finest teaching force."
+	icon_state = "traindrobe"
+	icon_lightmask = "base_drobe"
+	icon_panel = "drobe"
+	icon_broken = "base_drobe"
+	icon_off = "base_drobe"
+	category = VENDOR_TYPE_CLOTHING
+	ads_list = list("You're gonna LEARN today!", "What am I supposed to do?")
+	vend_reply = "Thank you for using the TrainDrobe!"
+	products = list(
+		/obj/item/clothing/gloves/color/white = 2,
+		/obj/item/clothing/gloves/color/black = 2,
+		/obj/item/clothing/shoes/laceup = 2,
+		/obj/item/clothing/shoes/black = 2,
+		/obj/item/clothing/shoes/brown = 2,
+		/obj/item/clothing/head/drilltrainer = 2,
+		/obj/item/clothing/under/rank/procedure/nct = 2,
```

Comment:
```suggestion
		/obj/item/clothing/under/rank/procedure/nct = 2,
		/obj/item/clothing/under/rank/procedure/nct/skirt = 2,
```

Skirt currently isn't available in the vending machine

Reply:
Also might want to add the plasmaman items to the vending machine as well as per this PR https://github.com/ParadiseSS13/Paradise/pull/27255